addons/sourcemod/scripting/include/surftimer.inc

```diff
@@ -117,7 +118,7 @@ native int surftimer_GetMapData(char[] name, char[] time);
  * @param countryCode			String to store the client's countryCode.
  * @param continentCode			String to store the client's continentCode.
  */
-native void surftimer_GetPlayerData(int& client, char[] time, int& rank, char[] country, char countryCode[3], char continentCode[3]);
+native void surftimer_GetPlayerData(int& client, float& time, int& rank, char[] country, char countryCode[3], char continentCode[3]);
```

Comment:
What's the reason for this change? Plugins which use the old version, will not work anymore.

Reply:
having the value itself seems to be a better practice, it can be manipulated much easier instead of begin restricted to a formatted string